### ISSUE TITLE
GeoFence

### ISSUE THREAD

**Issue by Tixav** [hansworst24/Ticz#23]
Just to write an idea to add to ticz, a geofence option that could turn on/off a dummy switch

**Comment by hansworst24** [comment]
Yeah, thought about that, but it would require some background task running / polling for location and depending on that trigger something in Ticz... From what I've read, Windows Phone does a poor job in running background tasks, so updates would be quite unstable... So I leave it for the moment as a feature request :)
